lib/commentOnOpenedPullRequest.js

```diff
@@ -32,6 +32,19 @@ function commentOnOpenedPullRequest(body, repositorySettings) {
     return commentOnOpenedPullRequest._implementation(filesUrl, commentsUrl, repositorySettings, userName, repositoryUrl, baseBranch, headBranch);
 }
 
+/**
+ * Transform a GitHub API URL to a regular GitHub URL.
+ *
+ * @param {String} apiURL The GitHub API URL.
+ * @returns {String} A regular GitHub URL.
+ */
+function getURLFromAPIUrl(apiURL) {
```

Comment:
So we use the contributors file for both linking to the file itself, and pulling the content via the GitHub API to check the list, correct?

I think the string in the config file should be the relative file path (ie. `CONTRIBUTORS.md`) in the repo, not the API URL. That was it should be easier to construct the proper URL for each purpose. 

Also, add a test to make sure this is working for both forks and branches. 

For a branch the URL should be `https://github.com/<OWNER>/<REPO>/blob/<BRANCH>/<PATH>`.
For a fork, the URL should be `https://github.com/<USER>/<REPO>/blob/<BRANCH>/<PATH>`.